doc/installation/windows.md

```diff
@@ -24,3 +24,21 @@ C:\> pip install --pre --no-cache --extra-index-url https://mssqlcli.blob.core.w
 ```shell
 C:\> pip uninstall mssql-cli
 ```
+
+## Instalation behind a proxy

```

Comment:
@bigleka, thanks for the contribution! Would you mind moving this section to under the daily preview build install section? Thanks!

Reply:
@bigleka , also can you fix typo. Thanks!